module/VuFind/tests/unit-tests/src/VuFindTest/OAuth2/Repository/AbstractTokenRepositoryTestCase.php

```diff
@@ -209,8 +167,8 @@ protected function createAccessTokenEntity(array $fields): AccessTokenEntityInte
         $mock->method('getUser')->willReturnCallback(function () use ($i) {
             $userId = $this->accessTokenTable[$i]['user_id'] ?? null;
             if ($userId) {
-                $userTable = $this->getMockUserTable();
-                return $userTable->getById($userId);
+                $userTable = $this->getMockUserService()->getUserByField('id', $userId);
+                return $userTable;
```

Comment:
I think this is more clear if we don't introduce a $userTable variable.

```suggestion
                return $this->getMockUserService()->getUserByField('id', $userId);
```